src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    private static final int GROW_CONSTANT = 2;
+
+    private List<Node<K, V>>[] buckets;
```

Comment:
Actually, i think that your solution will work properly, but why do we need List array, when we can just create Node array like in original implementation of HashMap?
```suggestion
    private Node<K, V>[] buckets;
```
Then we could avoid adding extra lines of code and improve the overall understanding of it

Reply:
yeah, it's works, but u are right, so i remove all it and start all over again